Altis_Life.Altis/core/pmenu/fn_keyDrop.sqf

```diff
@@ -3,24 +3,42 @@
     Author: Bryan "Tonic" Boardwine
 
     Description:
-    Drops the key.
+    Drops a key to a vehicle (Excluding houses).
 */
-private ["_dialog","_list","_sel","_vehicle","_impounded","_owners","_index","_index2","_i"];
 disableSerialization;
 
-_dialog = findDisplay 2700;
-_list = _dialog displayCtrl 2701;
-_sel = lbCurSel _list;
-if (_sel isEqualTo -1) exitWith {hint localize "STR_NOTF_noDataSelected";};
-if ((_list lbData _sel) isEqualTo "") exitWith {hint localize "STR_NOTF_didNotSelectVehicle";};
-_index = (parseNumber(_list lbData _sel));
-_vehicle = life_vehicles select _index;
-if (_vehicle isKindOf "House_F") exitWith {hint localize "STR_NOTF_cannotRemoveHouseKeys";};
-_owners = _vehicle getVariable "vehicle_info_owners";
-
-_index2 = [(getPlayerUID player),_owners] call TON_fnc_index;
+private _dialog = findDisplay 2700;
+private _list = _dialog displayCtrl 2701;
+private _sel = lbCurSel _list;
+
+if (_sel isEqualTo -1) exitWith {
+    hint localize "STR_NOTF_noDataSelected"
+};
+
+if (_list lbData _sel isEqualTo "") exitWith {
+    hint localize "STR_NOTF_didNotSelectVehicle"
+};
+
+private _index = parseNumber (_list lbData _sel);
+private _vehicle = life_vehicles param [_index, objNull, [objNull]];
+
+// Do not let them drop the key to a house
+if (_vehicle isKindOf "House_F") exitWith {
+    hint localize "STR_NOTF_cannotRemoveHouseKeys"
+};
+
+// Solve stupidness
+private _op = objectParent player;
+if (_op isEqualTo _vehicle && {locked _vehicle isEqualTo 2}) exitWith {
+    hint "You cannot drop keys to a vehicle which is locked and you are inside of."
```

Comment:
@SimZor, could you please [localise](https://community.bistudio.com/wiki/localize) the above string [under notifications in the stringtable](https://github.com/AsYetUntitled/Framework/blob/b525f34ad9fe5435b61649cb1f2a086efebad3d4/Altis_Life.Altis/stringtable.xml#L3102)?